task1/src/components/modal/modal.scss

```diff
@@ -0,0 +1,58 @@
+#modal {
+    .gray-back {
+        position: fixed;
+        top: 0;
+        left: 0;
+        right: 0;
+        bottom: 0;
+        background-color: rgba(0, 0, 0, .7);
+        z-index: 1000;
+    }
+    .modal-inner {
+        position: fixed;
+        top: 50%;
+        left: 50%;
+        transform: translate(-50%, -50%);
+        background-color: #FFF;
+        z-index: 1000;
+
+        width: 40rem;
+        // height: 50rem;
```

Comment:
can we remove this?

Reply:
removing now
(i hope minifier will do it for me)